src/themes/custom/app/login-page/login-page.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="container w-100 h-100">
```

Comment:
It's unclear to me why a `login-page.component.html` is being added to the `custom` theme in this PR.  Is this accidental?